### PR TITLE
Fixed the broken link

### LINKED ISSUE #101008: [Umbrella] Fix KEP link
``` 
grep -nr '/enhancements/keps/' * | grep -v '^vendor' | grep -v '/README.md' | grep -v '.json' | grep -v '.pb.go' | grep -v '.proto' | grep -v 'generated'
```

- [ ] OWNERS_ALIASES:581:  # conformance aliases https://git.k8s.io/enhancements/keps/sig-architecture/20190412-conformance-behaviors.md
#101009
- [x] pkg/apis/core/types.go:2835:	// More info: https://git.k8s.io/enhancements/keps/sig-network/0007-pod-ready%2B%2B.md
~#100996~ #99391
- [x] pkg/kubelet/pod/mirror_client.go:85:	// See http://git.k8s.io/enhancements/keps/sig-auth/20190916-noderestriction-pods.md
#101011
- [x] pkg/kubelet/config/flags.go:92:	fs.MarkDeprecated("redirect-container-streaming", "Container streaming redirection has been removed from the kubelet as of v1.20, and this flag will be removed in v1.22. For more details, see http://git.k8s.io/enhancements/keps/sig-node/20191205-container-streaming-requests.md")
#101037
- [x] staging/src/k8s.io/api/core/v1/types.go:3122:	// More info: https://git.k8s.io/enhancements/keps/sig-network/0007-pod-ready%2B%2B.md
#99391
- [x] staging/src/k8s.io/api/core/v1/types.go:3129:	// More info: https://git.k8s.io/enhancements/keps/sig-node/runtime-class.md
~#101014~ #99391
- [x] staging/src/k8s.io/api/node/v1beta1/types.go:36:// https://git.k8s.io/enhancements/keps/sig-node/runtime-class.md
#99391
- [x] staging/src/k8s.io/api/node/v1alpha1/types.go:34:// https://git.k8s.io/enhancements/keps/sig-node/runtime-class.md
#99391

```
grep -nr 'github.com/kubernetes/enhancements/' * | grep -v '^vendor' | grep -v '/README.md' | grep -v '/issues/' | grep -v '/pull/'  | grep -v '.json' | grep -v '.pb.go' | grep -v '.proto' | grep -v 'generated'
```

- [x] CHANGELOG/CHANGELOG-1.8.md:1434:[scalability validation report]: https://github.com/kubernetes/enhancements/tree/master/release-1.8/scalability_validation_report.md
#102860
- [x] CHANGELOG/CHANGELOG-1.14.md:943:kubernetes/enhancements: [kep](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/kubeadm/0029-20180918-kubeadm-phases-beta.md)
~#101140~ #102860
- [x] pkg/ssh/ssh.go:53: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] pkg/volume/util/metrics.go:41: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/kube-aggregator/pkg/controllers/status/metrics.go:27: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/legacy-cloud-providers/gce/token_source.go:46: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go:31: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go:57: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go:29: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go:36: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/metrics.go:36: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics.go:26: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/audit/metrics.go:35: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [x] staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509.go:38: * ALPHA stability level https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes)
#101046
- [ ] test/conformance/testdata/ineligible_endpoints.yaml:108:  link: https://github.com/kubernetes/enhancements/blob/f16c4c7f1c9e28a3cc4bb4d0e6503efea2ae7987/keps/sig-api-machinery/20190228-priority-and-fairness.md
#101138
- [ ] test/conformance/testdata/ineligible_endpoints.yaml:111:  link: https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190802-dynamic-coordinated-storage-version.md
#101138

Some needs to be regenerated `make update`

### PR DESCRIPTION
**Pull request by BodilessSleeper**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

Fixes #101008

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by BodilessSleeper** [comment]
I have signed the document

**Review by ehashman — COMMENTED** [review]
/kind cleanup
/ok-to-test
/triage accepted
/priority backlog
/lgtm

File: pkg/kubelet/config/flags.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'remote'.")
 	fs.StringVar(&s.RuntimeCgroups, "runtime-cgroups", s.RuntimeCgroups, "Optional absolute name of cgroups to create and run the runtime in.")
 	_ = fs.Bool("redirect-container-streaming", false, "[REMOVED]") // TODO: Delete in v1.22
-	fs.MarkDeprecated("redirect-container-streaming", "Container streaming redirection has been removed from the kubelet as of v1.20, and this flag will be removed in v1.22. For more details, see http://git.k8s.io/enhancements/keps/sig-node/20191205-container-streaming-requests.md")
+	fs.MarkDeprecated("redirect-container-streaming", "Container streaming redirection has been removed from the kubelet as of v1.20, and this flag will be removed in v1.22. For more details, see https://git.k8s.io/enhancements/keps/sig-node/1558-streaming-proxy-redirects/README.md")
```
Comment (wzshiming) [inline pkg/kubelet/config/flags.go:92]: I noticed that it needs to be removed in v1.22, Please delete L91-92

Comment (BodilessSleeper) [reply pkg/kubelet/config/flags.go]: Done

**Comment by BodilessSleeper** [comment]
Sry for the ping in advance @wzshiming

**Pull request merged**